### DISCUSSION TITLE
Column animation

### DISCUSSION THREAD

**Discussion by barbinbrad** [TanStack/table#4613] [Q&A]
Hey guys, first of all thank you so much to everyone involved in making this!

I just migrated to v8 from v7, and it's great. One thing I noticed is that there is no animated example. I tried implementing it similarly to v7 -- but without luck. Any insights into whether it _should_ work or what would need to be done to get it to work?

**Comment by barbinbrad** [comment]
It works.

**Comment by andrewnaeve** [comment]
Is there anything you can share about how you implemented?

**Reply by barbinbrad** [reply]
Sure it's a lot like the example from v7 (framer motion wrapped `tr`, `td`, `th` inside an `AnimatePresence`:

https://github.com/barbinbrad/carbon/blob/54e4fb23c8075af970495fcb55c95142876cab16/apps/carbon/app/components/Table/Table.tsx#L486-L494

I ended up removing this because it wasn't playing nicely with virtual scrolling.
